readme.txt

```diff
@@ -82,147 +103,5 @@ This should be a path (i.e. `/test`) or a URL (i.e. `http://example.com/wp/test`
-* Bug fix: Don't perform redirects in the admin. Props [joshbetz](https://github.com/joshbetz).
-* Bug fix: Prevent duplicate GUIDs. Props [danblaker](https://github.com/danblaker).
-
-= 1.7.1 - 2013-12-12 =
-* Add 307 redirect status code. Thanks [lgedeon](https://github.com/lgedeon)
-* Plugin textdomain should be loaded on init
-* Add status code labels to creation dropdown. Thanks Chancey Mathews
-
-= 1.7 - 2013-04-06 =
-* Return redirect_from on get_permalink. Thanks [simonwheatley](https://github.com/simonwheatley)
-* Allow for regex replacement in from/to redirects
-* Add Slovak translation. Thanks [Branco Radenovich](http://webhostinggeeks.com/blog)
-* Notice fixed in filter_admin_title
-
-= 1.6 - 2012-12-11 =
-* Bulk delete redirects from the Manage Redirects screen
-* wp-cli coverage including subcommands for creating, deleting, and listing redirects, and importing .htaccess files
-
-= 1.5 - 2012-11-07 =
-* Regular expressions allowed in redirects
-* New filter 'srm_registered_redirects' allows you to conditionally unset redirects based on context, user permissions, etc. Thanks [jtsternberg](https://github.com/jtsternberg) for the pull request.
-
-= 1.4.2 - 2012-10-17 =
-* Disable redirect loop checking by default. You can filter srm_check_for_possible_redirect_loops to enable it.
-* Only return published redirects in update_redirect_cache. - bug fix
-
-= 1.4.1 - 2012-10-11 =
-* Refresh cache after create_redirect call - bug fix
-* Refresh cache after save_post is called - bug fix
-* Chop off "pre-WP" path from requested path. This allows the plugin to work on WP installations in sub-directories - bug fix
-
-= 1.4 - 2012-10-09 =
-* Use the '*' wildcard at the end of your match value to configure a wildcard redirect. Use the same symbol at the end of your redirect to value in order to have the matched value be appended to the end of the redirect. Thanks [prettyboymp](https://github.com/prettyboymp) for the pull request
-* Change default request-matching behavior to be case-insensitive. This can be modified using the 'srm_case_insensitive_redirects' filter.
-* Include an informational 'X-Safe-Redirect-Manager' header when a redirect occurs. Thanks [simonwheatley](https://github.com/simonwheatley) for the pull request
-
-= 1.3 - 2012-09-19 =
-* safe-redirect-manager.php - Globalize SRM class for use in themes/plugins/scripts. Added create_redirect method to make importing easier.
-
-= 1.2 - 2012-09-01 =
-*   safe-redirect-manager.php - manage_options capability required to use redirect manager, remove checkbox column, hide view switcher, fix search feature, hide privacy stuff for bulk edit
-
-= 1.1 - 2012-08-28 =
-*   safe-redirect-manager.php - plugin_url() used properly, is_plugin_page function
-
-= 1.0 - 2012-08-27 =
-*   Plugin released
-
-== Upgrade Notice ==
-
-= 2.0.0 =
-Note that this version bumps the PHP minimum from 5.6 to 7.4 and the WordPress minimum from 4.6 to 5.7.
+= Earlier versions =
```

Comment:
I'm not sure how the WP.org readme.txt parser will handle this section but let's YOLO into it and see.  If it comes out oddly formed then we can do a change just to the readme on trunk to update on WP.org

Reply:
@jeffpaul [WordPress SEO](https://wordpress.org/plugins/wordpress-seo/#developers) is using same. 
https://github.com/Yoast/wordpress-seo/blob/00eb29a34edd4db41288eecc63a588fb9f1a46cb/readme.txt#L379